eng/mgmt/mgmtmetadata/alertsmanagement_resource-manager.txt

```diff
@@ -3,13 +3,18 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/alertsmanagement/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-2021-08 --csharp-sdks-folder=C:\Repos\sdk\azure-sdk-for-net\sdk
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/alertsmanagement/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-2021-08 --csharp-sdks-folder=C:\Repos\last_sdk_comment\azure-sdk-for-net\sdk
```

Comment:
Please follow the [introduction](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec)
to update:
1. Update the following information for the .csproj(not test .csproj)
    Version (Must for new nuget package)
    PackageTags (Must for new API version)
    PackageReleaseNotes (this is important because this information is displayed on [www.nuget.org](http://www.nuget.org/)  when your nuget package is published)
2. Update AzSdk.RP.props
    AzureApiTag: This will be displayed as Nuget package tag
3. Update the following information in AssemblyInfo.cs
    AssemblyFileVersion (Must have)
    AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed